### PR TITLE
Adds icon for MTN Mobile Money wallet

### PR DESCRIPTION
**Pull request by tolu-paystack**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by tolu-paystack** [comment]
Hi @tauthomas01 could you check this one out too? Thanks

**Comment by tolu-paystack** [comment]
Hi @tauthomas01 could you help here please?

File: app/assets/images/payment_icons/MTN.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...5.5 0 01-.36-.56l1.06-5.89v-.39l-.17-.07-1.14 6.3a.79.79 0 00.56.89l3.43 1a.85.85 0 00.22 0 .82.82 0 00.31-.07l.62-.26a.78.78 0 00.47-.6l1.71-10a.79.79 0 00-.22-.69"/><path d="M25.31 12.48a.23.23 0 00.17.25l1 .25a.21.21 0 00.26-.15l.08-.43a.23.23 0 00-.18-.25l-1-.24a.19.19 0 00-.24.15zM23.42 12a.22.22 0 00.16.24l.92.24a.19.19 0 00.24-.15l.08-.41a.23.23 0 00-.17-.25l-.92-.22a.18.18 0 00-.23.14zm1.64 1.65a.23.23 0 00.17.25l1 .27a.19.19 0 00.25-.14l.08-.43a.23.23 0 00-.18-.26l-1-.26a.19.19 0 00-.24.14zm-1.89-.56a.22.22 0 00.16.25l.92.25a.18.18 0 00.24-.14l.08-.41a.23.23 0 00-.17-.25l-.91-.25a.2.2 0 00-.24.14zm1.65 1.73a.24.24 0 00.17.26l1 .29a.19.19 0 00.25-.14l.08-.43a.25.25 0 00-.18-.27l-1-.27a.19.19 0 00-.24.14zm-1.88-.58a.23.23 0 00.16.25l.91.27a.19.19 0 00.24-.14l.08-.42a.22.22 0 00-.16-.25l-.92-.26a.18.18 0 00-.23.13z"/><path class="c" d="M22.17 3.43L20.15 7c.86.12 5.67 1.26 6.65 3.54a1.74 1.74 0 01.1.23v-.25a12.14 12.14 0 00-.9-4.11 5.75 5.75 0 00-.36-.65 5.1 5.1 0 00-3.43-2.35"/><path class="a" d="M23.49 8.09a2 2 0 10-2-2 2 2 0 002 2"/><path class="c" d="M25.17 6.05a1.68 1.68 0 11-1.68-1.68 1.68 1.68 0 011.68 1.68m.21 0a1.89 1.89 0 10-1.89 1.89 1.89 1.89 0 001.89-1.89"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/MTN.svg:1]: Can we replace all `class` by using the values (`fill: #fff`, `fill:#ffcb05`, etc) directly in the svg code?

Comment (tolu-paystack) [reply app/assets/images/payment_icons/MTN.svg]: @tauthomas01 done!

File: app/assets/images/payment_icons/MTN.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...-.56l1.06-5.89v-.39l-.17-.07-1.14 6.3a.79.79 0 00.56.89l3.43 1a.85.85 0 00.22 0 .82.82 0 00.31-.07l.62-.26a.78.78 0 00.47-.6l1.71-10a.79.79 0 00-.22-.69"/><path d="M25.31 12.48a.23.23 0 00.17.25l1 .25a.21.21 0 00.26-.15l.08-.43a.23.23 0 00-.18-.25l-1-.24a.19.19 0 00-.24.15zM23.42 12a.22.22 0 00.16.24l.92.24a.19.19 0 00.24-.15l.08-.41a.23.23 0 00-.17-.25l-.92-.22a.18.18 0 00-.23.14zm1.64 1.65a.23.23 0 00.17.25l1 .27a.19.19 0 00.25-.14l.08-.43a.23.23 0 00-.18-.26l-1-.26a.19.19 0 00-.24.14zm-1.89-.56a.22.22 0 00.16.25l.92.25a.18.18 0 00.24-.14l.08-.41a.23.23 0 00-.17-.25l-.91-.25a.2.2 0 00-.24.14zm1.65 1.73a.24.24 0 00.17.26l1 .29a.19.19 0 00.25-.14l.08-.43a.25.25 0 00-.18-.27l-1-.27a.19.19 0 00-.24.14zm-1.88-.58a.23.23 0 00.16.25l.91.27a.19.19 0 00.24-.14l.08-.42a.22.22 0 00-.16-.25l-.92-.26a.18.18 0 00-.23.13z"/><path d="M22.17 3.43L20.15 7c.86.12 5.67 1.26 6.65 3.54a1.74 1.74 0 01.1.23v-.25a12.14 12.14 0 00-.9-4.11 5.75 5.75 0 00-.36-.65 5.1 5.1 0 00-3.43-2.35" fill="#00678f"/><path d="M23.49 8.09a2 2 0 10-2-2 2 2 0 002 2" fill="#fff"/><path d="M25.17 6.05a1.68 1.68 0 11-1.68-1.68 1.68 1.68 0 011.68 1.68m.21 0a1.89 1.89 0 10-1.89 1.89 1.89 1.89 0 001.89-1.89" fill="#00678f"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/MTN.svg:1]: Hi ,

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- We are missing the `<title>` attribute

Comment (tolu-paystack) [reply app/assets/images/payment_icons/MTN.svg:1]: @tauthomas01 so sorry about that. Is that the only outstanding issue? Just want to make sure I'm not missing anything else so we can close this out

Comment (tolu-paystack) [reply app/assets/images/payment_icons/MTN.svg]: Added the title attribute

Comment (tauthomas01) [reply app/assets/images/payment_icons/MTN.svg]: Looks great!

**Comment by tauthomas01** [comment]
The rest looks good! Can you rebase off master branch?
After that I will merge it.

**Comment by tauthomas01** [comment]
Hi @tolu-paystack ,

I can't merge your branch because you need rebase off master branch.

**Comment by tolu-paystack** [comment]
Hey @tauthomas01. I was actually gonna wait until you were able to take a look at #281, just so I know I rebased properly. Then I'll make the necessary changes for this branch too

**Pull request closed without merging**